codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -492,13 +476,12 @@ public void onGoAwayRead(ChannelHandlerContext ctx, int lastStreamId, long error
         public void onWindowUpdateRead(ChannelHandlerContext ctx, int streamId, int windowSizeIncrement)
                 throws Http2Exception {
             Http2Stream stream = connection.requireStream(streamId);
-            verifyGoAwayNotReceived();
-            if (stream.state() == CLOSED || shouldIgnoreFrame(stream, false)) {
-                // Ignore frames for any stream created after we sent a go-away.
+
+            if (stream.state() == CLOSED || streamCreatedAfterGoAwaySent(stream)) {
```

Comment:
@Scottmitch I don't get it. A WINDOW_UPDATE is not a flow controlled frame. So why count it towards flow control?